components/vf-hero/vf-hero--example.njk

```diff
@@ -0,0 +1,48 @@
+<style>
+  :root {
+    --vf-hero-bg-image: url('{{ '../../assets/vf-hero/assets/vf-intro-group.png' | path }}');
+  }
+</style>
+
+
+<section class="vf-hero vf-hero--easy">
```

Comment:
I'm assuming these would be variants but just haven't been broken out for the first iteration?

Reply:
Yep.